core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -82,7 +82,7 @@ object ReassignPartitionsCommand extends Logging {
     }
   }
 
-  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions) {
+  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions): Map[TopicAndPartition, Seq[Int]] = {
```

Comment:
OK. I like that approach too and it will simply my JUnit tests. I will borrow that in this PR. You still need to resolve the conflict through in the future.